js/renovation/ui/button.tsx

```diff
@@ -74,20 +74,20 @@ export const viewFunction = (viewModel: Button) => {
       {...viewModel.restAttributes} // eslint-disable-line react/jsx-props-no-spreading
     >
       <div className="dx-button-content" ref={viewModel.contentRef as any}>
-        {template
-                && (
-                <viewModel.props.template
-                  data={{ icon, text }}
-                />
-                )}
-        {!template && children}
+        {ButtonTemplate && (<ButtonTemplate data={{ icon, text }} />)}
+        {!ButtonTemplate && children}
```

Comment:
Why is template replaced to ButtonTemplate?

Reply:
JSX requires upper case name for the component